### PR TITLE
Fix/run one master cluster

### PR DESCRIPTION
**Pull request by bxy4543**

### CODE REVIEW

File: apply/run.go [hunk 1]
```diff
@@ -40,22 +41,27 @@ func IsIPList(args string) bool {
...
-		c.cluster.Spec.SSH.Passwd = c.passwd
-		c.cluster.Spec.Provider = common.AliCloud
-		return nil
+	c.cluster.Spec.Image = c.imageName
+	if IsNumber(c.masterArgs) {
+		if IsNumber(c.nodeArgs) || c.nodeArgs == "" {
+			c.cluster.Spec.Masters.Count = c.masterArgs
+			c.cluster.Spec.Nodes.Count = c.nodeArgs
+			c.cluster.Spec.SSH.Passwd = c.passwd
+			c.cluster.Spec.Provider = common.AliCloud
```
Comment (oldthreefeng) [inline apply/run.go:50]: c.cluster.Spec.Provider need to read from Cluster.  or set a default CloudProvider
If user want to Use other Cloud . we may suport for multi CloudProvider soon?  

```
DefaultCloudProvider = AliCloud
common.DefaultCloudProvider
```

**Review by oldthreefeng — COMMENTED** [review]
LGTM. just Run a single master cluster will couse the error.

File: apply/run.go (hunk 1 above)
Comment (cuisongliu) [reply apply/run.go:50]: or add cloud provider , if masterArgs is number then replace the provider

File: apply/run.go [hunk 2]
```diff
@@ -14,6 +14,7 @@ import (
 
 type ClusterArgs struct {
 	cluster    *v1.Cluster
+	imageName  string
```
Comment (cuisongliu) [inline apply/run.go:17]: this args should be from cmd/run.go transfer to here. I not found `imageName` args in cmd/run command, you not finished yet?

**Review by cuisongliu — DISMISSED** [review]
please add images args in cmd/run.go

File: apply/run.go [hunk 3]
```diff
@@ -40,22 +41,27 @@ func IsIPList(args string) bool {
...
 
 func (c *ClusterArgs) SetClusterArgs() error {
-	if IsNumber(c.masterArgs) && IsNumber(c.nodeArgs) {
-		c.cluster.Spec.Masters.Count = c.masterArgs
-		c.cluster.Spec.Nodes.Count = c.nodeArgs
-		c.cluster.Spec.SSH.Passwd = c.passwd
-		c.cluster.Spec.Provider = common.AliCloud
-		return nil
+	c.cluster.Spec.Image = c.imageName
+	if IsNumber(c.masterArgs) {
```
Comment (fanux) [inline apply/run.go:45]: if isnumber(master) && (isNUmber(node) || nodeargs == "")

**Review by fanux — APPROVED** [review]

**Pull request merged**